repo2docker/buildpacks/conda/freeze.py

```diff
@@ -118,13 +126,26 @@ def set_python(py_env_file, py):
 
 
 if __name__ == "__main__":
-    # allow specifying which Pythons to update on argv
-    pys = sys.argv[1:] or ("2.7", "3.6", "3.7", "3.8")
+    parser = ArgumentParser()
+    parser.add_argument("--containerrun", default="docker", help="Container runtime")
+    parser.add_argument("--volopts", default="Host volume mount options")
```

Comment:
```suggestion
    parser.add_argument("--volopts", help="Host volume mount options")
```
should it also have a default value?

Reply:
I thought argparse defaults to `""`? I can make it explicit though.

Reply:
I'm confused. Currently you are setting the default value of this argument to `"Host volume mount options"`. This seems like it should be the help text, not the value. I hadn't thought of the default value of `default=`

Reply:
Sorry, you're right! I made an error, and also managed to misread your comment.